### PR TITLE
VCR split

### PR DESCRIPTION
**Pull request by shuyama1**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by shuyama1** [comment]
Added `affected service packages` in the test report. see https://github.com/GoogleCloudPlatform/magic-modules/pull/9392#issuecomment-1861110946

Already cleaned up the cassettes recorded for testing this PR change.

**Review by trodge — CHANGES_REQUESTED** [review]

**Review by melinath — APPROVED** [review]
LGTM overall

File: .ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh [hunk 1]
```diff
@@ -108,61 +108,67 @@ fi
 
 update_status "pending"
 
-TF_LOG=DEBUG TF_LOG_PATH_MASK=$local_path/testlog/replaying/%s.log TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test $GOOGLE_TEST_DIRECTORY -parallel $ACCTEST_PARALLELISM -v -run=TestAcc -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google-beta/version.ProviderVersion=acc" > replaying_test.log
-
-test_exit_code=$?
-
-TESTS_TERMINATED=$(grep "^cannot run Terraform provider tests" replaying_test.log)
-
-counter=1
```
[the remark this answers is not in the document]
Comment (shuyama1) [reply .ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh:117]: When VCR was initially set up in GCB, we used to run into replaying tests got terminated due to non-provider-related issues, so we rerun the tests if that error occurred (with max retry: 3) example: https://github.com/GoogleCloudPlatform/magic-modules/pull/5881#issuecomment-1086319609

The issue should be resolved after we pre-install Terraform, but we never removed the logic. I've never seen any VCR test rerun needed for replaying mode after https://github.com/GoogleCloudPlatform/magic-modules/pull/6034, so I think it should be safe to remove it.

I agree it's unrelated - I could keep it for now and remove it in a separate PR.

[the remark this answers is not in the document]
Comment (melinath) [reply .ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh:117]: I'm fine with keeping this in

File: .ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh [hunk 2]
```diff
@@ -108,61 +108,67 @@ fi
...
-test_suffix=""
+run_full_VCR=false
+declare -A affected_services
+
+for file in $gofiles
+do
+  if [[ $file = google-beta/services* ]]; then
+    affected_services[$(echo "$file" | awk -F / '{ print $3 }')]=1
```
[the remark this answers is not in the document]
Comment (shuyama1) [reply .ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh:122]: Added. Let me know if any part is still unclear and needs more comments. Thanks:)

**Review by trodge — APPROVED** [review]

**Pull request merged**